cmd/img_mgmt/port/zephyr/src/zephyr_img_mgmt.c

```diff
@@ -36,6 +36,18 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include <img_mgmt/image.h>
 #include "../../../src/img_mgmt_priv.h"
 
+static int
+slot_to_image_index(int slot)
```

Comment:
Did not find a good macro for this. Does it exsists?

Reply:
No there is not.